src/posts/xrisk.md

```diff
@@ -90,14 +85,12 @@ An AI could have any goal.
 Maybe it wants to calculate pi, maybe it wants to cure cancer, maybe it wants to self-improve.
 This depends on how it is trained.
 But even though we cannot tell what a superintelligence will want to achieve, we can make predictions about its sub-goals.
-We can [mathematically prove](https://arxiv.org/pdf/1912.01683.pdf) what a rational machine will pursue in _virtually any goal_:
```

Comment:
Btw, this paper has been dis-endorsed by its own author as irrelevant.

Reply:
Didn't know that, looks like I've got some reading to do. 

This post is probably a good start: https://www.lesswrong.com/posts/pdaGN6pQyQarFHXF4/reward-is-not-the-optimization-target